From e0b142cfec431a78ffcb9db051b2f21423fe88d1 Mon Sep 17 00:00:00 2001
From: Varun Kumar Reddy Yaparla <vyaparla@habana.ai>
Date: Mon, 27 Dec 2021 18:33:45 +0200
Subject: [PATCH] [SW-67662] gaudi2: change fake security spi offset

This change modifies SPI flash offset where fake security data
is stored from 2 MB offset to 3 MB. This is due to issue in spi driver
where data is not properly written at 2 MB offset

Change-Id: Ieda5d9b5a91730839a9f78f48556902770283454
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index c205648..b7ba9c6 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -314,8 +314,8 @@
 
 #define SZ_1K			1024
 #define SZ_1M			(SZ_1K * SZ_1K)
-#define FAKE_OTP_OFFSET		(2 * SZ_1M)
-#define FAKE_FUSE_OFFSET	(2 * SZ_1M + 64 * SZ_1K)
+#define FAKE_OTP_OFFSET		(3 * SZ_1M)
+#define FAKE_FUSE_OFFSET	(3 * SZ_1M + 64 * SZ_1K)
 
 static const struct hl_sec_efuse *preboot_get_hl_sec_fake_efuse(void)
 {
